csharp/Platform.IO/StreamExtensions.cs

```diff
@@ -16,6 +32,22 @@ public static void Write<T>(this Stream stream, T value)
             stream.Write(bytes, 0, bytes.Length);
         }
 
+        /// <summary>
+        /// <para>Reads a sequence of bytes representing the structure and moves the current position within the <paramref name="stream"/> by the number of bytes read.</para>
+        /// <para>Прочитывает последовательность байт представляющих структуру и перемещает текущую позицию в <paramref name="stream"/> вперед на число прочитанных байт..</para>
+        /// </summary>
+        /// <typeparam name="T">
+        /// <para>Type being structure.</para>
+        /// <para>Тип являющийся структурой.</para>
+        /// </typeparam>
+        /// <param name="stream">
+        /// <para>A stream containing the structure.</para>
+        /// <para>Поток, содержащий структуру.</para>
+        /// </param>
+        /// <returns>
+        /// <para>The structure, if all bytes from stream have been read, otherwise the structure with default fields values.</para>
```

Comment:
`otherwise the structure with default fields values` can be replaced with shorter version `otherwise the default value of <typeparamref name="T"/> structure`